### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Pacoscript**

### CODE REVIEW

File: staff/francisco-fernandez/app-template/about.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+        <h1>About Us</h1>
```
Comment (joel-molero) [inline staff/francisco-fernandez/app-template/about.html:13]: For better semantics, include ```<h1>```in the tag ```<header>```inside ```<body>```

File: staff/francisco-fernandez/app-template/about.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        <p>We are a new young company with a lot of illusion and no idea of programming</p>
```
Comment (joel-molero) [inline staff/francisco-fernandez/app-template/about.html:16]: Likewise, include ```<p>```inside a ```<main>```tag

File: staff/francisco-fernandez/app-template/about.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        <footer>September 2018</footer>
```
Comment (joel-molero) [inline staff/francisco-fernandez/app-template/about.html]: Inside ```<footer>```, try including a ```<time>```tag in order to administer better your data for future uses.

File: staff/francisco-fernandez/app-template/home.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Home Page</title>
+</head>
+<body>
+    <section>
+    <h1>My Website</h1>
```
Comment (joel-molero) [inline staff/francisco-fernandez/app-template/home.html]: Include your links inside a ```<nav>```tag for better semantics and maybe inside an ```<ul>```or ```<ol>```

**Review by joel-molero — CHANGES_REQUESTED** [review]

File: staff/francisco-fernandez/app-template/home.html (hunk 4 above)
Comment (aleixcam) [reply staff/francisco-fernandez/app-template/home.html]: and the anchors inside ```<li>``` too

File: staff/francisco-fernandez/app-template/contact.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <h3>Contact Form</h3>
```
Comment (bcasasus) [inline staff/francisco-fernandez/app-template/contact.html]: If `<h3>Contact Form</h3>` is the page title would be nice to wrap it with a `<header>` tag. If it is the `<form>` title the should be inside the form tag.

**Form example:**
```
<form action="" method="post">
  <fieldset>
    <legend>Title</legend>
    <input type="radio" name="radio" id="radio">
    <label for="radio">Click me</label>
  </fieldset>
</form>
```

**Further information:**

- [MDN Header Tag](https://developer.mozilla.org/ca/docs/Web/HTML/Element/header)

- [MDN Use of Heading Elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)

File: staff/francisco-fernandez/app-template/about.html (hunk 1 above)
Comment (Pacoscript) [reply staff/francisco-fernandez/app-template/about.html:13]: done

File: staff/francisco-fernandez/app-template/about.html (hunk 2 above)
Comment (Pacoscript) [reply staff/francisco-fernandez/app-template/about.html:16]: done, thanks

File: staff/francisco-fernandez/app-template/contact.html (hunk 5 above)
Comment (Pacoscript) [reply staff/francisco-fernandez/app-template/contact.html]: done, thanks

**Pull request merged**